packages/ecc-client-ga4gh-tes/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "ecc-client-ga4gh-tes",
+  "description": "Web componets for TES",
+  "main": "index.js",
+  "private": true,
+  "type": "module",
+  "version": "0.1.0",
+  "author": "",
+  "license": "ISC",
+  "repository": "",
+  "scripts": {
+    "dev": "tsc && concurrently -k -r \"tsc --watch --preserveWatchOutput\"",
+    "lint:fix": "npm run lint -- --fix",
+    "lint": "eslint --ext .ts,.html . --ignore-path .gitignore && prettier \"**/*.ts\" --check --ignore-path .gitignore",
+    "format": "eslint --ext .ts,.html . --fix --ignore-path .gitignore && prettier \"**/*.ts\" --write --ignore-path .gitignore",
+    "test": "tsc && wtr --coverage",
```

Comment:
It `test` working?

Reply:
FAST's test wasn't working; this one does, though since atm I don't have a test file, I have to create a dummy .test.ts to commit code otherwise this works afaik and wrt my last tries.